app/build-types/mmi/manifest/_base.json

```diff
@@ -0,0 +1,26 @@
+{
+  "browser_action": {
+    "default_icon": {
+      "16": "images/icon-16.png",
+      "19": "images/icon-19.png",
+      "32": "images/icon-32.png",
+      "38": "images/icon-38.png",
+      "64": "images/icon-64.png",
+      "128": "images/icon-128.png",
+      "512": "images/icon-512.png"
+    },
+    "default_title": "MetaMask Institutional"
+  },
+  "icons": {
+    "16": "images/icon-16.png",
+    "19": "images/icon-19.png",
+    "32": "images/icon-32.png",
+    "38": "images/icon-38.png",
+    "48": "images/icon-48.png",
+    "64": "images/icon-64.png",
+    "128": "images/icon-128.png",
+    "512": "images/icon-512.png"
+  },
+  "name": "__MSG_appNameMmi__",
```

Comment:
The message `appNameMmi` seems to be missing from our localizations. See the `appNameBeta` message for an example of what we need here (we'll want it to be present in every single locale file, and we don't need it to be translated).

Chrome will reject any extension as invalid if a referenced message is missing in any locale.

Reply:
Oh! ok, we thought we could add them all later in a new/specific PR, ok we will have to add them as we go then 👍🏼 thanks!